dev/02_dev.R

```diff
@@ -183,6 +184,10 @@ usethis::use_data_raw( name = "adsl", open = FALSE ) # not run.
 ## Tests ---- not run
 ## Add one line by test you want to create
 usethis::use_test( "app" )
+usethis::use_test("assessmentInfoServer")
+
+golem::add_module( name = "big_fake" ) # ran
+golem::use_module_test("big_fake")
 
 # Documentation
 ## Vignette ---- notrun
```

Comment:
Looks like a commit from another PR snuck it's way in here.

Reply:
Whoops!

Reply:
There is also the line `usethis::use_package("rjson", type = "Suggests")